src/sentry/workflow_engine/models/workflow_group_status.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+
+
+@region_silo_model
+class WorkflowGroupStatus(DefaultFieldsModel):
```

Comment:
Is the plan here to use `date_updated` as the equivalent of `last_active`?

Reply:
yes, is that possible/wise? i believe you can manually update it with `update()`

Reply:
I think it's ok... we'd only really be updating it in one case. Maybe just test locally (write a throwaway test?) to see if you can do the equivalent of  
```
        updated = (
            GroupRuleStatus.objects.filter(id=status.id)
            .exclude(last_active__gt=freq_offset)
            .update(last_active=now)
        )
 ```